Marlin/src/pins/stm32f4/pins_BTT_BTT002_V1_0.h

```diff
@@ -27,7 +27,7 @@
   #error "BIGTREE BTT002 V1.0 supports up to 1 hotends / E-steppers."
 #endif
 
-#define BOARD_INFO_NAME "BIGTREE Btt002 1.0"
+#define BOARD_INFO_NAME "BTT Btt002 V1.0"
```

Comment:
Since we're already changing names... let's capitalize BTT002: "BTT BTT002 V1.0"